src/store/modules/images.js

```diff
@@ -259,6 +266,42 @@ const actions = {
      *     taken the active user or not.
      */
 
+  // we want to group images as they load and we group them based on their SMARTSTK value
+  // we dispatch this action in multiple places
+  group_images ({ commit, state, rootState, dispatch }) {
+    const currentSite = rootState.site_config.selected_site
+    let grouping_images = JSON.parse(JSON.stringify(state.grouped_images))
```

Comment:
Nitpick, but can we rename `grouping_images` to `grouped_images`? I think it's clear that the variable has local scope and won't be confused with the global state variable.

Reply:
Renamed to `grouped_images_local` since it can't be named grouped_images because that already exists